components/IssuesTable.tsx

```diff
@@ -58,11 +59,20 @@ export default function IssuesTable(props: Props) {
         ),
         defaultCanSort: false
       },
+      {
+        Header: 'Start Date',
+        id: 'startDate',
+        accessor: 'project_in_progress_at',
+        className: 'w-32',
+        Cell: ({row, cell}: CellProps<Card, Date>) => (
+          <TargetDate targetDate={cell.value} />
```

Comment:
We probably should renabe this component as ReportDate. Not a blocker.

Reply:
Pushed a change to rename this to `ReprotDate`.